pkg/pipeline/source/sdk.go

```diff
@@ -88,11 +88,13 @@ func NewSDKSource(ctx context.Context, p *config.PipelineConfig, callbacks *gstr
 			s.startRecording.Break()
 		}),
 	}
-	if p.AudioTempoController.Enabled {
-		// perform signal time comression/steatching instead of timestamp manipulation
-		// on RTCP sender reports
-		logger.Debugw("audio tempo controller enabled", "adjustment rate", p.AudioTempoController.AdjustmentRate)
+	if p.RequestType == types.RequestTypeRoomComposite || p.AudioTempoController.Enabled {
+		// in case of room composite don't adjust audio timestamps on RTCP sender reports,
+		// to avoid gaps in the audio stream
 		opts = append(opts, synchronizer.WithAudioPTSAdjustmentDisabled())
+		if p.AudioTempoController.Enabled {
+			logger.Debugw("audio tempo controller enabled", "adjustment rate", p.AudioTempoController.AdjustmentRate)
```

Comment:
:nit: I have found it easier logging keys without spaces for filtering, something like `adjustmentRate` in this case.